host/sgx/sgxload.c

```diff
@@ -476,12 +496,12 @@ oe_result_t oe_sgx_create_enclave(
     }
 
     /* Create SECS structure */
-    if (!(secs = _new_secs(
-              (uint64_t)base,
-              enclave_size,
-              oe_sgx_is_debug_load_context(context))))
+    if (!(secs = _new_secs((uint64_t)base, enclave_size, context)))
         OE_RAISE(OE_OUT_OF_MEMORY);
 
+    /* Set the platform xfrm value in context */
```

Comment:
This is unnecessary now and circular in logic. Please remove.

Reply:
Oops, Good catch. Removing.